include/stddef.h

```diff
@@ -20,7 +20,9 @@ typedef _PDCLIB_ptrdiff_t ptrdiff_t;
 typedef _PDCLIB_size_t size_t;
 #endif
 
+#ifndef __cplusplus
 typedef _PDCLIB_wchar_t   wchar_t;
+#endif
```

Comment:
This might also be suitable for upstream.

Even if pdclib doesn't want C++ compatibility, it probably doesn't want C++ incompatibility either.

Reply:
Almost every .h has
```
#ifdef __cplusplus
extern "C"
#endif
```
which leads me to believe that they actually do want C++ compatibility. :)